docs/Clearwater_stress_testing.md

```diff
@@ -12,82 +13,112 @@ package installed.
-6. Follow [this process](https://github.com/Metaswitch/crest/blob/dev/docs/Bulk-Provisioning%20Numbers.md) to bulk provision subscribers. Create 30000 subscribers per SIPp node.
-7. Create your stress test node by typing `knife box create -E ENVIRONMENT sipp --index 1`.  If you have multiple bono nodes, you'll need to create multiple stress test nodes by repeating this command with "--index 2", "--index 3", etc. - each stress test node only sends traffic to the bono with the same index.
-  * To create multiple nodes, try `for x in {1..20} ; do { knife box create -E ENVIRONMENT sipp --index $x && sleep 2 ; } ; done`.
-  * To modify the number of calls/hour to simulate, edit/add `count=<number>` to `/etc/clearwater/shared_config`, then run `sudo /usr/share/clearwater/infrastructure/scripts/sip-stress` and `sudo service clearwater-sip-stress restart`.
-8. Create a Cacti server for monitoring the deployment, as described in [this document](Cacti.md).
-9. When you've finished, destroy your deployment with `knife deployment delete -E ENVIRONMENT`.
+Total calls: 1625
+Successful calls: 1612 (99.2%)
+Failed calls: 13 (0.8%)
 
-### Manual (i.e. non-Chef) stress runs
+Retransmissions: 0
 
-SIP stress can also be run against a deployment that has been installed manually (as per the [Manual Install instructions](Manual_Install.md)).
+Average time from INVITE to 180 Ringing: 74.0 ms
+# of calls with 0-2ms from INVITE to 180 Ringing: 0 (0.0%)
+# of calls with 2-20ms from INVITE to 180 Ringing: 0 (0.0%)
+# of calls with 20-200ms from INVITE to 180 Ringing: 1572 (96.7384615385%)
+# of calls with 200-2000ms from INVITE to 180 Ringing: 45 (2.76923076923%)
+# of calls with 2000+ms from INVITE to 180 Ringing: 0 (0.0%)
 
-Firstly follow [this process](https://github.com/Metaswitch/crest/blob/dev/docs/Bulk-Provisioning%20Numbers.md) to bulk provision subscribers. Work out how many stress nodes you want, and create 30000 subscribers per SIPp node.
+Total re-REGISTERs: 1800
+Successful re-REGISTERs: 1792 (99.5555555556%)
+Failed re-REGISTERS: 8 (0.444444444444%)
 
-To create a new SIPp node, create a new virtual machine and bootstrap it [by configuring access to the Project Clearwater Debian repository](Manual_Install.md#configure-the-apt-software-sources).
+REGISTER retransmissions: 0
 
-Then set the following properties in /etc/clearwater/local_config:
+Average time from REGISTER to 200 OK: 15.0 ms
 
-* (required) local_ip - the local IP address of this node
-* (optional) node_idx - the node index (defaults to 1)
+Log files in /var/log/clearwater-sip-stress
+```
 
-Set the following properties in /etc/clearwater/shared_config:
+The summary statistics at the end just come from SIPp, not the Clearwater deployment. If you want to monitor Sprout CPU during the stress run, you'll need to be running a separate monitoring tool such as Cacti (and we provide the start and end time of the stress run, to let you match up with these external graphs).
```

Comment:
The summary statistics at the end just come from SIPp, not the Clearwater deployment. *If you need more detailed statistics... blah blah, link to stats page